;;; TOOL: run-interp-spec
;;; STDIN_FILE: third_party/testsuite/data.wast
(;; STDOUT ;;;
out/test/spec/data.wast:162: assert_unlinkable passed:
  error: data segment is out of bounds: [0, 1) >= max value 0
  0000016: error: OnDataSegmentData callback failed
out/test/spec/data.wast:170: assert_unlinkable passed:
  error: data segment is out of bounds: [0, 1) >= max value 0
  0000017: error: OnDataSegmentData callback failed
out/test/spec/data.wast:178: assert_unlinkable passed:
  error: data segment is out of bounds: [0, 1) >= max value 0
  0000017: error: OnDataSegmentData callback failed
out/test/spec/data.wast:186: assert_unlinkable passed:
  error: data segment is out of bounds: [1, 1) >= max value 0
  0000015: error: OnDataSegmentData callback failed
out/test/spec/data.wast:194: assert_unlinkable passed:
  error: data segment is out of bounds: [1, 1) >= max value 0
  0000016: error: OnDataSegmentData callback failed
out/test/spec/data.wast:211: assert_unlinkable passed:
  error: data segment is out of bounds: [666, 667) >= max value 0
  0000030: error: OnDataSegmentData callback failed
out/test/spec/data.wast:220: assert_unlinkable passed:
  error: data segment is out of bounds: [65536, 65537) >= max value 65536
  0000019: error: OnDataSegmentData callback failed
out/test/spec/data.wast:227: assert_unlinkable passed:
  error: data segment is out of bounds: [65536, 65537) >= max value 65536
  0000029: error: OnDataSegmentData callback failed
out/test/spec/data.wast:235: assert_unlinkable passed:
  error: data segment is out of bounds: [131072, 131073) >= max value 131072
  0000018: error: OnDataSegmentData callback failed
out/test/spec/data.wast:243: assert_unlinkable passed:
  error: data segment is out of bounds: [131072, 131073) >= max value 131072
  0000019: error: OnDataSegmentData callback failed
out/test/spec/data.wast:251: assert_unlinkable passed:
  error: data segment is out of bounds: [4294967295, 4294967296) >= max value 65536
  0000016: error: OnDataSegmentData callback failed
out/test/spec/data.wast:258: assert_unlinkable passed:
  error: data segment is out of bounds: [4294967295, 4294967296) >= max value 65536
  0000027: error: OnDataSegmentData callback failed
out/test/spec/data.wast:266: assert_unlinkable passed:
  error: data segment is out of bounds: [4294967196, 4294967197) >= max value 131072
  0000017: error: OnDataSegmentData callback failed
out/test/spec/data.wast:273: assert_unlinkable passed:
  error: data segment is out of bounds: [4294967196, 4294967197) >= max value 65536
  0000028: error: OnDataSegmentData callback failed
out/test/spec/data.wast:283: assert_invalid passed:
  000000b: error: data section without memory section
out/test/spec/data.wast:292: assert_invalid passed:
  0000013: error: expected i32 init_expr
out/test/spec/data.wast:300: assert_invalid passed:
  0000014: error: expected END opcode after initializer expression
out/test/spec/data.wast:308: assert_invalid passed:
  0000012: error: unexpected opcode in initializer expression: 1 (0x1)
out/test/spec/data.wast:316: assert_invalid passed:
  0000012: error: unexpected opcode in initializer expression: 1 (0x1)
out/test/spec/data.wast:324: assert_invalid passed:
  0000014: error: expected END opcode after initializer expression
20/20 tests passed.
;;; STDOUT ;;)
